manke/htcondor/multus-nad-htcondor.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Multus has already been migrated to fleet.  The NAD should go here: https://github.com/lsst-it/k8s-cookbook/tree/master/fleet/lib/multus-conf/overlays/manke

The filename should be `net-attach-def-htcondor`.  Note that most of the existing file are network-attachment-definition and these should be renamed to follow the convention of using the kind's "short name", if one is set (e.g. `kubectl api-resources`).

Reply:
See #301